internal/services/sockets/server.go

```diff
@@ -0,0 +1,10 @@
+package sockets
+
+import "net/http"
+
+func NewServer() *http.Server {
+	return &http.Server{
```

Comment:
I propose we put the websocket logic in this package. I'm not a huge fan of this package name `sockets`, seems too generic, but naming is hard and I just wanted to get something started.